module/mempool/stdmap/chunk_requests.go

```diff
@@ -0,0 +1,70 @@
+package stdmap
```

Comment:
This is not a new implementation, I basically did a `git mv` for `engine/verification/fetcher/chunks.go` and then did some (major) refactorings.